pkgs/tools/misc/expect/default.nix

```
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
 
   hardeningDisable = [ "format" ];
 
-  postPatch = ''
+  preConfigure = ''
     sed -i "s,/bin/stty,$(type -p stty),g" configure
```

Doing this still sounds more natural to me:

```suggestion
  postPatch = ''
    sed -i "s,/bin/stty,$(type -p stty),g" configure.in
```

Ideally we would fix this upstream by adding `--with-stty` configure flag or something.